src/neptune/handler.py

```diff
@@ -595,7 +596,11 @@ def delete_files(self, paths: Union[str, Iterable[str]], *, wait: bool = False)
         return self._pass_call_to_attr(function_name="delete_files", paths=paths, wait=wait)
 
     @check_protected_paths
-    def download(self, destination: str = None) -> None:
+    def download(
+        self,
+        destination: str = None,
```

Comment:
`destination` should have type `Optional[str]` if you are allowed to pass None.
Not sure why mypy is passing on that

Reply:
Interesting. I see that it's been here for a long time, so yeah - weird. Worth looking into that. Thanks for noticing! Won't hurt to correct it anyway I guess, so let me do that quickly ;)